packages/testkit-backend/src/feature/common.js

```diff
@@ -33,7 +33,7 @@ const features = [
   'Feature:API:ConnectionAcquisitionTimeout',
   'Feature:API:Driver:GetServerInfo',
   'Feature:API:Driver.VerifyConnectivity',
-  'Feature:API:Result.Peek',
```

Comment:
why was this one removed?

Reply:
Because reactive doesn't support Peek. This feature is enable for async at `packages/testkit-backend/src/feature/async.js`